app/helpers/admin/taggable_content_helper.rb

```diff
@@ -55,6 +55,22 @@ def taggable_ministerial_role_appointments_container
     end
   end
 
+  def taggable_needs_container
+    Rails.cache.fetch("need.linkables", expires_in: 1.minute) do
+      publishing_api_with_low_timeout = Services.publishing_api.dup.tap do |client|
```

Comment:
> Just curious, what's the reasoning for caching this?

As @elenatanasoiu says, I think I copied it from the code that works with Policies, and caching will help with reliability.

https://github.com/alphagov/whitehall/blob/64549b845d58e3d4f60509f3b42ef88f93f4d728/app/models/policy.rb#L44-L57